### ISSUE TITLE
Is it possible to filter bag files by more than one parameter?

### ISSUE THREAD

**Issue by BaltashovIlia** [swri-robotics/bag-database#94]
Is it possible to filter bag files by more than one parameter? For example, to filter bag files that have two set topics or a set topic and a tag.

**Comment by connoranderson** [comment]
I do not believe this is currently possible. The only way I have found to search multiple tags is to combine both search terms into the same tag so that a single search query can search across both parameters at the same time.

**Comment by pjreed** [comment]
Yeah, right now that is not possible -- I have ideas in my head about implementing a search mechanism that lets you construct arbitrary boolean expressions, I just haven't done it.  It would be nice, though!
